app/templates/route-not-found.hbs

```diff
@@ -0,0 +1,12 @@
+{{page-title "404 - Pagina niet gevonden"}}
+
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-region-large">
+    <div class=" au-o-layout">
+      <AuContent @skin="large">
+        <h1>We kunnen de pagina niet vinden</h1>
```

Comment:
Just a question. When should the AuContent component be used instead the AuHeading one for example? I assumed it was needed to style dynamic / user generated content but since it's used here I don't think it's that simple?

Reply:
Generally I use the AuHeading component for titles in the interface. I use the AuContent component when there is user-generated content or grouped text (titles and paragraphs). Benefit is that the AuContent component also has spacing set which helps when there is a combo of titles and paragraphs.